### DISCUSSION TITLE
Is it possible to create a resolver without a query?

### DISCUSSION THREAD

**Discussion by Jacob-Mahoney** [MichalLytek/type-graphql#931] [Q&A]
**Describe the Bug**
Is it possible to create a resolver without a query? Whenever I try the following code, I get an error.

**To Reproduce**
```
@Resolver()
export class UserResolver {

    private manager: UserManager;

    constructor() {
        this.manager = new UserManager();
    }

    @Mutation(returns => Number)
    async createUser(@Arg("input") createUserInput: CreateUserInput): Promise<number> {
        let createdUserId = await this.manager.createUser(createUserInput);
        return createdUserId;
    }

}
```

**Expected Behavior**
To **not** get the following error: `Type Query must define one or more fields.`

**Environment (please complete the following information):**

- OS: Windows
- Node version: 14.15.4
- Package version: 1.1.1
- TypeScript version: 4.3.4

**Comment by MichalLytek** [comment]
In GraphQL spec, there must be at least one Query defined.
So in TypeGraphQL, you must have at least one `@Resolver` with one `@Query` defined.
